src/header/header.css

```diff
@@ -1,6 +1,6 @@
 header {
   height: 2.5em;
-  background-color: #e85248;
+  background-color: rgb(235,51,0);
```

Comment:
<img width="327" alt="screen shot 2017-11-29 at 17 55 18" src="https://user-images.githubusercontent.com/18505817/33390675-8826698c-d52e-11e7-8f83-0adda26fa78e.png">
Here are the values from the design manual